packages/tre/src/plugins/core/index.ts

```diff
@@ -412,10 +377,13 @@ export const CORE_PLUGIN: Plugin<
           cacheApiOutbound: { name: getCacheServiceName(workerIndex) },
         },
       });
-      serviceEntryBindings.push({
-        name: BINDING_SERVICE_USER,
-        service: { name },
-      });
+    } else if (workerIndex === 0 || options.routes?.length) {
+      throw new MiniflareCoreError(
+        "ERR_ROUTABLE_NO_SCRIPT",
+        `Worker [${workerIndex}] ${
+          options.name === undefined ? "" : `("${options.name}") `
+        }must have code defined as it's routable or the fallback`
+      );
```

Comment:
Why can any workers have no code? What does it mean for a worker to not have code defined?